app/src/main/java/com/example/tour_a_versity_v3/ui/MapsActivity.kt

```diff
@@ -13,13 +14,15 @@ import com.google.android.gms.maps.model.MarkerOptions
 
 class MapsActivity : AppCompatActivity(), OnMapReadyCallback, GoogleMap.OnInfoWindowClickListener {
 
+    private val DEFAULT_ZOOM_LEVEL = 17f
```

Comment:
A candidate for being a top-level constant declaration.

Reply:
Eventually, we will want to allow the user to change the zoom level. I agree the default zoom can be a constant, but then we will need to set up a way to allow the user to zoom in or out either with a button or gestures.

Reply:
In that case you can have another variable somewhere in the class that controls the current zoom level. In the beginning you can assign it to this default value, then later if the user zooms in/out, or you programmatically change the zoom level, you can change this current zoom variable.